src/DataEditor.js

```diff
@@ -37,7 +37,7 @@ export default class DataEditor extends PureComponent<
 
   render() {
     const { getValue, column, row, cell } = this.props;
-    const value = getValue({ column, row, data: cell }) || "";
+    const value = getValue({ column, row, data: cell });
```

Comment:
???

Reply:
I moved this condition up to be an inner part of `getValue()`
look my response for the `src/utils.js` it's part of the same change i made